Task2/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("        Task2");
+            Console.WriteLine("     variable type\n");
+            int a= 12;
+            Int32 b= 13;
+            byte c= 14;
+            Byte d= 15;
+            long e= 16;
+            Int64 f= 17;
+            char g= 'k';
+            Char i= 'i';
+            object k= 20;
+            Object l=29;
+            string o= "erer";
+            String p= "sdl";
+            
+
+            Type type_a= a.GetType();

```

Comment:
in C# better use the following naming - 

type_a  => `typeA`
type_b  => `typeB`